src/indev/lv_indev.c

```diff
@@ -806,9 +806,6 @@ static void indev_encoder_proc(lv_indev_t * i, lv_indev_data_t * data)
     lv_group_t * g = i->group;
     if(g == NULL) return;
 
-    indev_obj_act = lv_group_get_focused(g);
-    if(indev_obj_act == NULL) return;
-
```

Comment:
Keep this one instead will keep functionality unchanged. Though I don't know if we should use second statement.

Reply:
I would also keep this one.